src/main/resources/db/migration/R__TypeCodes.sql

```diff
@@ -265,6 +270,11 @@ VALUES (1, 'SeedBank'),
        (3, 'PlantingSite')
 ON CONFLICT (id) DO UPDATE SET name = excluded.name;
 
+INSERT INTO tracking.mangrove_tides (id, name)
+VALUES (1, 'High'),
+       (2, 'Low')
```

Comment:
Possibly silly nitpick, but would it be better if "low" had a lower ID than "high" rather than the other way around?